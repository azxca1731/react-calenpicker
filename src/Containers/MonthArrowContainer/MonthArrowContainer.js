import React from "react";
import PropTypes from "prop-types";

import { CssConnector } from "Containers/Provider";
import MontArrow from "Components/MonthArrow";

const MonthArrowContainer = props => <MontArrow {...props} />;

MonthArrowContainer.propTypes = {
  type: PropTypes.oneOf(["left", "right"]).isRequired,
  onClick: PropTypes.func.isRequired,
  cssObject: PropTypes.object,
  theme: PropTypes.string
};

export default CssConnector(({ state }) => ({
  cssObject: state.MonthArrowCssObject
}))(MonthArrowContainer);
