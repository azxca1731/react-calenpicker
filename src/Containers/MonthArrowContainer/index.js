import MonthArrowContainer from "./MonthArrowContainer";

export default MonthArrowContainer;
