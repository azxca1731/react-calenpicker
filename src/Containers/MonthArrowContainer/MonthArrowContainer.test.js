import React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";

import MonthArrowContainer from "./index";

describe("MonthArrowContainer", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<MonthArrowContainer />);
    expect(wrapper).toHaveLength(1);
  });

  it("renders with snapshot", () => {
    const wrapper = shallow(<MonthArrowContainer />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
