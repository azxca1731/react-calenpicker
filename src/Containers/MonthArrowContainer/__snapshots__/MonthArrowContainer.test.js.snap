// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MonthArrowContainer renders with snapshot 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
